cmake/modules/BuildQAT.cmake

```diff
@@ -0,0 +1,55 @@
+function(build_qat)
+  set(QAT_REPO https://github.com/intel/qatlib.git)
```

Comment:
earlier generations of QAT devices (e.g. c62x, dh895xxcc, etc.) are not supported by in-tree qat driver and [qatlib](https://github.com/intel/qatlib).